interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -112,14 +120,30 @@ private void parseArgs(String[] args) {
       }
     }
 
+    if (secureMode == (port == maintenancePort)) {
```

Comment:
Do we want to enforce this for the insecure mode? If so, we need to handle the case when `--port` is set to something that isn't `8080` via CLI args, otherwise it'll break existing runners. Let's say one of existing runners overrides the port:

`bin/xds-test-server --port=8081`  

It'll compare port `8081` to the default maintenance port `8080`. So it'll enter this if and try to enforce the security mode.

Reply:
That's a good point. I'll change this to:
```
    if (secureMode && (port != maintenancePort)) {
      System.err.println(
          "port and maintenancePort should be different for secure mode: port="
              + port
              + ", maintenancePort="
              + maintenancePort);
      usage = true;
    }
```
For the insecure mode if maintenancePort is specified, it will be just ignored but no need to catch or flag that use-case